application/controllers/admin/PrintableSurvey.php

```diff
@@ -238,7 +247,7 @@ public function index($surveyid, $lang = null, $bReturn = false)
                                 // cqid == 0  ==> token attribute match
                                 $tokenData = getTokenFieldsAndNames($surveyid);
                                 preg_match('/^{TOKEN:([^}]*)}$/', (string) $distinctrow['cfieldname'], $extractedTokenAttr);
-                                $sExplanation .= "Your " . $tokenData[strtolower($extractedTokenAttr[1])]['description'] . " ";
+                                $sExplanation .= "Your " . ($tokenData[strtolower($extractedTokenAttr[1])]['description'] ?? "") . " ";
```

Comment:
If this is null : it's an issue with your Survey data …

Reply:
@Shnoulle you may be correct, but the page should nevertheless not crash even then.